requirements/main.in

```diff
@@ -5,16 +5,22 @@
 # After editing, update requirements/main.txt by running:
 #     make update-deps
 
-# prevent a resolution conflict
-# uvicorn 0.19 wants h11 0.14.0, but httpcore 0.15.0 (which is ancient, but demanded by httpx)
-# demands h11 0.12.0
-h11==0.12.0
-alembic~=1.8
-astropy~=5.1
-asyncpg~=0.26
+# Resolve a dependency issue by forcing a version of packaging
```

Comment:
Very minor comment, but you're mixing comments that don't (like this line) and that do (like line 18) end in a `.`. Please pick one and stick to that.

Reply:
Standard python style is to write full sentences with period.